puzzle/cli/store.py

```diff
@@ -0,0 +1,253 @@
+import os
+import logging
+import click
+
+from . import (cli, root, family_file, family_type, variant_type, mode,
+ get_home_dir)
+
+from puzzle.plugins import SqlStore, VcfPlugin
+try:
+    from puzzle.plugins import GeminiPlugin
+except ImportError:
+    pass
+
+from sqlite3 import DatabaseError
+
+
+logger = logging.getLogger(__name__)
+
+@cli.command()
+@click.option('--reset', is_flag=True)
+@root
+@click.pass_context
+def init(ctx, reset, root):
+    """Initialize a database that store metadata
+
+        Check if "root" dir exists, otherwise create the directory and
+        build the database. If a database already exists, do nothing.
+
+        The behaviour will be different with different plugins. A config file
+        in YAML format will be created in puzzle/configs with information about
+        the database.
+
+        VCF:
+            A sqlite database will be built in the home directory of the user
+        GEMINI:
+            A sqlite database will be built in the home directory of the user
+    """
+    root = root or ctx.obj['root']
```

Comment:
Let's make this a real code review ;)

This is overwriting the `root` import which is possibly confusing. I'd suggest renaming the import or alias it to: `from . import root as root_opt` - what do you think?